CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.5.0 (September 10, 2020)
+
+BUG FIXES:
+
+* wavefront_user resource fixed so that terraform doesn't try to constantly update the user groups field.
```

Comment:
Might want to add the PR number to this for tracking.

Reply:
Thanks Done.